README.md

```diff
@@ -20,35 +20,55 @@ k0s is an all-inclusive Kubernetes distribution with all the required bells and
 - Supports custom Container Network Interface (CNI) plugins (calico is the default)
 - Supports x86_64 and arm64
 
-## Quick start
+## Try k0s
 
-```
-curl -sSLf get.k0s.sh | sh
-k0s server --enable-worker
-```
+If you'd like to try k0s, please jump to our:
 
+- [Super QuickStart](#super_quickstart) - Create a k0s control plane and worker, and access it locally with kubectl.
+- [NanoDemo](#nanodemo) - Watch a .gif recording of how to create a k0s instance.  
+- [Better QuickStart](#better_quickstart) - Create a k0s control plane and worker, then access it with Lens from anywhere.
+- [Create a k0s cluster](https://github.com/k0sproject/k0s/blob/main/docs/create-cluster.md) - For when you're ready to build a multi-node cluster.
 
-![k0s demo](k0s_demo.gif)
+You may also be interested in [current version specifications](#version_specs). For docs, tutorials, and other k0s resources, see our [Docs and Resources](https://github.com/k0sproject/k0s/tree/main/docs/docs_resources_TOC) mainpage.
+
+## Join the Community
 
-## Motivation
+If you'd like to help build k0s, please check out our guide to [Contributing](https://github.com/k0sproject/k0s/tree/main/CONTRIBUTING.md) and our [Code of Conduct](https://github.com/k0sproject/k0s/tree/main/CODE_OF_CONDUCT.md).
+
+## What's Our Motivation?
+
+Why are we building k0s? Several reasons:
 
 **Note:** Some of these goals are not 100% fulfilled yet.
 
 _We have seen a gap between the host OS and Kubernetes that runs on top of it: How to ensure they work together as they are upgraded independent from each other? Who’s  responsible for vulnerabilities or performance issues originating from the host OS that affect the K8S on top?_
 
-**&rarr;** k0s Kubernetes is fully self contained. It’s distributed as a single binary with no host OS deps besides the kernel. Any vulnerability or perf issues may be fixed in k0s Kubernetes.
+**&rarr;** k0s Kubernetes is fully self contained. It’s distributed as a single binary with no host OS dependencies besides the kernel. Any vulnerability or performance issues can be fixed in k0s Kubernetes.
 
 _We have seen K8S with partial FIPS security compliance: How to ensure security compliance for critical applications if only part of the system is FIPS compliant?_
 
-**&rarr;** k0s Kubernetes core + all included host OS dependencies + components on top may be compiled and packaged as a 100% FIPS compliant distribution with a proper toolchain.
+**&rarr;** k0s Kubernetes core + all included host OS dependencies + components on top may be compiled and packaged as a 100% FIPS-compliant distribution with a proper toolchain.
```

Comment:
```suggestion
**&rarr;** k0s core, all host OS dependencies and the components on top may be compiled and packaged as a 100% FIPS-compliant distribution using a proper toolchain.
```

"distribution with a proper toolchain" would be a distribution that has a proper toolchain.